examples/misc/lib/effective_dart/usage_good.dart

```diff
@@ -338,14 +338,15 @@ class Treasure {
 }
 
 class C {
-  double left, right, top, bottom, minTime;
+  num left, right, top, bottom, minTime;
```

Comment:
Did you mean to turn this from `double` to `num`? 

(I'm actually surprised it's not `int` here...)

Reply:
Oops, missed your comments here before I committed. I don't recall why I changed these. I think there was an analysis error. I'll check.